### PR TITLE
Feat: Added Pages & routes

### PR DESCRIPTION
**Pull request by anamkaa**
**Added Pages**
- Home Page
- Note-taking Page
- Archive Page
-  Trash Page
- Label Page
- Signup Page
- Login Page

https://user-images.githubusercontent.com/75376715/183451353-b04dc34e-c5f7-4acd-b42e-fc57a5866b8d.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,25 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
-import logo from "./logo.png";
+import ArchivePage from "./pages/ArchivePage";
+import HomePage from "./pages/HomePage";
+import LabelPage from "./pages/LabelPage";
+import LoginPage from "./pages/LoginPage";
+import NoteTakingPage from "./pages/NoteTakingPage";
+import SignupPage from "./pages/SignupPage";
+import TrashPage from "./pages/TrashPage";
```
Comment (MaruthiPothuganti) [inline src/App.js]: Suggestion: Barrel exports will be clean

**Review by MaruthiPothuganti — COMMENTED** [review]
Everything looks good. PR can be merged.

**Review by hrshmistry — APPROVED** [review]
Well structured 👍🏻 go ahead

File: src/App.js (hunk 1 above)
Comment (anamkaa) [reply src/App.js]: Added barrel exports

**Pull request merged**